src/commands/send-request.js

```diff
@@ -28,9 +29,12 @@ const parseHttpFile = async (filePath, variables, rawMode) => {
   logger.debug(`Attempting to read request data from file: ${filePath}`);
   try {
     let fileContent = await Deno.readTextFile(filePath);
+    const secretKeys = config.getSecretKeys();
```

Comment:
Revert the concept of hiding secrets. It has to be done differently on logger level.